examples/webgl_loader_sea3d_morph.html

```diff
@@ -187,6 +182,9 @@
 
 				requestAnimationFrame( animate );
 
+				// Update SEA3D Animations
+				THREE.SEA3D.AnimationHandler.update( delta );
```

Comment:
It sounds like a good idea. But how make this? Mixer appears to be based on models like:

```javascript
new THREE.AnimationMixer( mesh )
new THREE.AnimationMixer( skinnedMesh )
..
new THREE.AnimationMixer( light )
```